stevensStringLib.h

```diff
@@ -1148,8 +1148,39 @@ namespace stevensStringLib
         return std::equal(str.begin(), str.begin() + str.size()/2, str.rbegin());
     }
 
-
-    //replace
+    /**
+    * Replaces characters using a vector syntax. Use multi_type_vector to construct proper input.
+    * Example syntax:
+    * 
+    * std::string testing = stevensStringLib::replace_characters({"testing {0}{1}", "1", "This is another test"}); // replaces string selection with specified characters
+    * std::cout << testing << std::endl;
+    * 
+    * @param vector<str> - example_vector above
+    * @retval str - returns replaced string
+    */
+    std::string replace_characters( std::vector<std::string> values ) 
+    {
+        std::string result = values[0];  // setting intital input
```

Comment:
I'll need to account for the possibility of someone passing in an empty values vector here.